client/pwa/src/service-worker.ts

```diff
@@ -25,34 +26,37 @@ const UPDATES_BASE_URL = `https://updates.${
   location.hostname === "localhost" ? "developer.allizom.org" : location.host
 }`;
 
+const SW_TYPE: SwType =
+  SwType[new URLSearchParams(location.search).get("type")] || SwType.ApiOnly;
```

Comment:
the left side of is returning a type (e.g. `ApiOnly`) and the right side is returning a number (e.g. `2`). I think `SwType.ApiOnly` needs to change into `SwType[SwType.ApiOnly]`

Reply:
Yes. I had string enums there and switched to numeric. Reverting that.